### ISSUE TITLE
javascript error on lib/scripts/collection.js

### ISSUE THREAD

**Issue by heartnetkung** [mongo-express/mongo-express#330]
I found out that this line caused the client javascript to throw and stopped working.
```js
renderjson.set_show_to_level(ME_SETTINGS.collapsibleJSONDefaultUnfold);
```
Somehow renderjson is {} as printed in console.log.
I got it to work by removing that line.

More info:
- version 0.39.1
- page viewing collection
- my setup: simply `npm install mongo-express` then run `node app.js`

I'll try to manage a pull request sometime later. For now, I'll just put it here in case someone found the same error.

**Comment by heartnetkung** [comment]
@dozoisch I've figured it out but this time I can't send you a pull request since it's a problem when you publish npm module. The problem is that the compressed javascript file `build/collection-xxx.min.js` is corrupted only when obtained from the npm. Here is the diff.

![diff](https://cloud.githubusercontent.com/assets/1158685/25062257/7af0433a-21f3-11e7-99e1-fa33dcbcaf83.png)

This problem only occurs in 0.39.1.

**Comment by dozoisch** [comment]
Wow, weird that it did that.  I'll build another version today. Thanks for investigating.

**Comment by dozoisch** [comment]
Should be fixed in v0.39.2
